sockets.js

```diff
@@ -212,7 +216,41 @@ if (cluster.isMaster) {
 	}
 
 	let app = require('http').createServer();
-	let appssl = Config.ssl ? require('https').createServer(Config.ssl.options) : null;
+	let appssl = null;
+	if (Config.ssl) {
+		let key;
+		if (Buffer.isBuffer(Config.ssl.options.key)) {
```

Comment:
In that casee this would assume the key/cert's a filename and never launch the HTTPS server. I was making a dumb assumption going off Go does it and not being able to properly test it. Probably should have asked you about it or something

Going off the documentation for the HTTPS module, it'd be simpler to check if `key` or `cert` are paths instead of checking for each possible type they can be, but I want to check if that's reasonable before rewriting this a third tiime

Reply:
Yeah, that's reasonable.

Reply:
Still waiting on this.

Reply:
....?

Reply:
OH
Damn it mrof